scripts/dev_scripts/integration_tests.sh

```diff
@@ -133,6 +133,18 @@ $RUN_MACARON analyze -rp https://github.com/apache/maven -b master -d 6767f2500f
 
 python $COMPARE_DEPS $DEP_RESULT $DEP_EXPECTED || log_fail
 
+
+echo -e "\n----------------------------------------------------------------------------------"
+echo "apache/maven: Analyzing using a CycloneDx SBOM with PURL."
+echo -e "----------------------------------------------------------------------------------\n"
+SBOM_FILE=$WORKSPACE/tests/dependency_analyzer/cyclonedx/resources/apache_maven_root_sbom_minimal.json
+DEP_EXPECTED=$WORKSPACE/tests/dependency_analyzer/expected_results/apache_maven_with_sbom_minimal.json
+DEP_RESULT=$WORKSPACE/output/reports/maven/org_apache_maven/maven/dependencies.json
+
+$RUN_MACARON analyze -purl pkg:maven/org.apache.maven/maven -sbom "$SBOM_FILE" || log_fail
```

Comment:
Is this new test supposed to be testing the new use case of no main target repository? If that is the intention, then that should be made more clear. Also, isn't that purl a real, valid purl that in principle could be resolved to a repo (so there could be a main target repo found)?

Reply:
You are right, this test case is used for testing the new use case of no main target repository. I worked on this test case back then before the feature for mapping PURL string to repository path of the repo-finder is implemented. I will change this to explicitly indicate that this is a non-repo PURL.

Reply:
Addressed in https://github.com/oracle/macaron/pull/165/commits/82ca751ab8b3e0ce79910ad413d05c4b8c16f07c